### ISSUE TITLE
Noises generated from !shaped-spells commands

### ISSUE THREAD

**Issue by mlenser** [symposion/roll20-shaped-scripts#457]
Is there a way to turn off chat noises for other players when running !shaped-spells commands?

**Comment by symposion** [comment]
So what exactly is happening here? Are other players getting the chat messages as well? Or just the noises? Which commands specifically are generating the noises - just the listing commands or the import commands as well?

**Comment by mlenser** [comment]
The messages are not seen, but the sound is generated from clicking a filter for example which generates a new list.

**Comment by symposion** [comment]
I can't reproduce this using two browsers with a friend's account. All  I'm doing is /w <Player Name> <List Code> so unless there's something broken with Roll20 this shouldn't happen. Could you try:

1. Reproducing in a new game with just two players
2. Seeing what happens if the player getting the issue does /w <Players Own Name> Foo in the original game that showed the problem

**Comment by mlenser** [comment]
It happened in a game with just 2 players. A player of mine was filtering spells with !shaped-spells and every time he made a filter selection I heard the chat bubble noise. I am GM in the game so I'm unsure if that makes a difference (it shouldn't).

I can test with him whispering himself, though I expect it to make no noises as I've never heard noises from other whispers before.

**Comment by symposion** [comment]
Ok I think I just reproduced this. I think only GMs get this when players are doing the spell list. I suspect this is going to turn out to be a roll20 bug unfortunately.

**Comment by mlenser** [comment]
I suspected it was a Roll20 issue, though I couldn't give proper detail for how you're sending the messages.

Could you open a bug report with them with the details of that format?

**Comment by symposion** [comment]
It's not that simple. It doesn't happen when I send a test whisper. I think it *may* be connected to the length of the message.

**Comment by symposion** [comment]
Yeah, there's something specific to the content of the spells list that it's not liking, but it's not obvious. This is going to take a while to debug - I'm going to have to come back to this after I've sorted out some other stuff. Sorry :-(

**Comment by mlenser** [comment]
Ok, no worries.

**Comment by symposion** [comment]
Aaargh. This is worse than I thought. It doesn't *always* happen even for !shaped-spells. I've spent a while trying to nail it down to no avail. I've raised https://app.roll20.net/forum/post/4912143/incorrect-slash-unwanted-chat-beeping-from-api-whispers/?pageforid=4912143#post-4912143 with what I've found, but without a reliable reproduction path I very much doubt it will get fixed. Sorry, not sure what more I can do here.

**Comment by mlenser** [comment]
This is a trivial issue so I wouldn't spend more time on it. It's only GMs hearing PC commands so it's not game breaking.
